### PR TITLE
WIP Adding flake8 to the project

### PR DESCRIPTION
**Pull request by eeshadutta**
I have corrected the linting errors in the code and have also added flake8 to the README.md file which contributors can run to fix PEP8 errors before commiting their changes.

### CODE REVIEW

**Comment by eeshadutta** [comment]
@madprime Please review and let me know if this is fine

File: ohapi/api.py [hunk 1]
```diff
@@ -132,17 +132,14 @@ def upload_file(target_filepath, metadata, access_token, project_member_id,
...
+    # .format(urlparse.urlencode({'access_token': access_token})))
 
     logging.info('Uploading {} ({})'.format(
         target_filepath, format_size(filesize)))
 
-    r = requests.post(url,
-                      files={'data_file': open(target_filepath, 'rb')},
-                      data={'project_member_id': project_member_id,
-                            'metadata': json.dumps(metadata)})
+    # r = requests.post(url, files={'data_file': open(target_filepath, 'rb')},
```
Comment (madprime) [inline ohapi/api.py]: This is removing functionality: `requests` is performing an action here, but storing the result is optional. So, to make `flake8` happy, the only thing you should remove is `r = `. Please update to fix this, thanks!

File: .flake8 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```
Comment (madprime) [inline .flake8]: I'd like to stick to the PEP8 recommendation. (I think it's 79... but probably you can just remove this file?.)

**Comment by eeshadutta** [comment]
@madprime I have made the required changes and have removed the .flake8 file.

**Pull request closed without merging**